HACKING.md

```diff
@@ -168,13 +130,14 @@ to 4.x:
 snap refresh snapcraft --channel=4.x/stable
 ```
 
-
 #### Splicing the snapd snap into the core snap
 
 Sometimes while developing you may need to build a version of the _core_ snap
-with a custom snapd version. The snapcraft.yaml for the core snap currently is
-complex in that it assumes it is built inside Launchpad with the 
-`snappy-dev/image` PPA enabled, so it is difficult to inject a custom version of
+with a custom snapd version. 
+The `snapcraft.yaml` for the [core snap](https://github.com/snapcore/core20/) 
```

Comment:
this link is wrong it should be to https://github.com/snapcore/core (and to be clear no, it is not updated to core20, that is a different snap, this is very specifically for the core snap which is special and distinct from the core20 snap)

Reply:
I don't think this was fixed

Reply:
I pushed a PR change the link https://github.com/snapcore/snapd/pull/11368